script/core/ActivateSemibook.s.sol

```diff
@@ -14,26 +14,37 @@ uint constant COVER_FACTOR = 1000;
   Activates a semibook on mangrove.
     outbound: outbound token
     inbound: inbound token,
-    outbound_in_gwei: price of one outbound token (display units) in gwei
+    outbound_in_gwei: price of one outbound token (display units) in gwei of native token
     fee: fee in per 10_000
 
   outbound_in_gwei should be obtained like this:
-  1. Get the price of one outbound token display unit in ETH
+  1. Get the price of one outbound token display unit in native token
   2. Multiply by 10^9
   3. Round to nearest integer*/
 
 contract ActivateSemibook is Test2, Deployer {
   function run() public {
+    Mangrove mgv_ = Mangrove(envAddressOrName("MGV", "Mangrove"));
+    (MgvStructs.GlobalPacked global,) = mgv_.config(address(0), address(0));
+
     innerRun({
       mgv: Mangrove(envAddressOrName("MGV", "Mangrove")),
+      gasprice: global.gasprice(),
       outbound_tkn: IERC20(envAddressOrName("OUTBOUND_TKN")),
       inbound_tkn: IERC20(envAddressOrName("INBOUND_TKN")),
       outbound_in_gwei: vm.envUint("OUTBOUND_IN_GWEI"),
       fee: vm.envUint("FEE")
     });
   }
 
-  function innerRun(Mangrove mgv, IERC20 outbound_tkn, IERC20 inbound_tkn, uint outbound_in_gwei, uint fee) public {
+  function innerRun(
+    Mangrove mgv,
+    uint gasprice,
```

Comment:
Why allow the gasprice used in market activation to differ from Mangrove's gasprice? Seem it'll make it easy to open a market with inconsistent parameters?

Reply:
This is actually a way to override. `gasprice` is actually used for both bounty provision computation and density. It would totally make sense to require density to be higher/lower than the current gasprice.

Reply:
Since Solidity doesn't allow optional parameters, I suggest reverting the `innerRun` signature and instead allowing the override in a `innerRunWithOverrides` function where `gasprice` can be overridden.

Reply:
> Since Solidity doesn't allow optional parameters, I suggest reverting the innerRun signature and instead allowing the override in a innerRunWithOverrides function where gasprice can be overridden.

Addressed in the latest push